web/js/src/helpers.js

```diff
@@ -1,8 +1,21 @@
-import { nanoid } from 'nanoid';
+import { customAlphabet } from 'nanoid';
 
-// Generates a random password using and ensures a number is included
+// Generates a random password that always passes password requirements
 export function randomPassword(length = 16) {
-	return nanoid(length) + Math.floor(Math.random() * 10);
+	const uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ';
+	const lowercase = 'abcdefghijklmnopqrstuvwxyz';
+	const numbers = '0123456789';
+	const symbols = '!@#$%^&*()_+-=[]{}|;:",./<>?';
```

Comment:
I am a fan of removing " from the the allowed list it could cause issue with the bash backend.